tests/integration/conftest.py

```diff
@@ -36,7 +36,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--snap-channel",
         type=str,
-        default="1.24/stable",
+        default="1.25/candidate",
```

Comment:
I am not sure there is a better way of doing this currently as the rc is only in this single channel. We would want to change this to latest/edge when 1.25 is put there I would think